consensus/service/config/src/error.rs

```diff
@@ -27,16 +27,8 @@ pub enum Error {
     /// Mint configuration is not allowed for token id {0}
     MintConfigNotAllowed(TokenId),
 
-    /**
-     * Invalid mint configuration for token id {0}: must have at least one
-     * signer
-     */
-    NoSigners(TokenId),
-
-    /** Invalid mint configuration for token id {0}: signer set threshold
-     * exceeds number of signers
-     */
-    SignerSetThresholdExceedsSigners(TokenId),
+    /// Invalid signer set for token id {0}
```

Comment:
I'm slightly concerned users will have trouble figuring out what the issue is with their config. Might it make sense to include an additional field (enum or string) with some detail? Perhaps modify the `is_valid` method to `validate(&self) -> Result<(), SignerSetValidationError>`

Reply:
I don't think that is an issue - the validation is pretty trivial. Additionally, I don't think it would be super helpful in the nested/recursive V2 implementation, since it won't include information about which of the nested signer sets caused the issue (unless I add code specifically to track the level of nesting, or render the problematic signer set or something along those lines).